ui/frontend/index.tsx

```diff
@@ -14,6 +14,7 @@ import {
   editCode,
   enableFeatureGate,
   gotoPosition,
+  addCodeToBeginning,
```

Comment:
WDYT about calling this `addImport` (and similar changes elsewhere) to match the intended usage? That way, if we can ever have something more capable, we know it’s safe to change.

Reply:
Seems reasonable. I've renamed it.